babble/src/main/java/io/mosaicnetworks/babble/node/ConfigManager.java

```diff
@@ -209,34 +219,35 @@ public static void setRootDir(String sRootDir) {
         ConfigManager.sRootDir = sRootDir;
     }
 
-
-
-
     /**
      * Configure the service to create a new group using the default ports
      * @param moniker node moniker
-     * @param inetAddress the IPv4 address of the interface to which the Babble node will bind
      * @throws IllegalStateException if the service is currently running
      */
-    public String createConfigNewGroup(GroupDescriptor groupDescriptor, String moniker, String inetAddress) {
-        return createConfigNewGroup(groupDescriptor, moniker, inetAddress, BabbleConstants.BABBLE_PORT());
```

Comment:
In particular here - this is code that actually uses the babble port - it is the non-archive path.

Reply:
I've removed calls to the BabbleConstants class since the ConfigManager can be used in isolation from the UI components, it can/should be configured using method calls. My thinking is that the XML approach should only be used for configuring UI components (if necessary the UI components can then call methods to configure non UI classes)

I haven't exposed methods for configuring file names since they cannot be changed without changing the go code. The babble directory name is an exception, which is not configurable, however this can be changed in future if necessary.

Reply:
I am arguing from a refactored code base where I am not sure that holds. But for the time being I think the pragmatic approach is to approve as-is and have this discussion again as part of the refactor merge....